src/clients/mentions.js

```diff
@@ -32,6 +32,20 @@ const fetchUserName = (accessToken, userId) => {
     }
 
     let promiseToGetUser = getPrincipal(accessToken, userId)
+        .then(responseJson => {
+            // according to https://cimpress.slack.com/archives/C5GD70W9J/p1565168815238400,
```

Comment:
This is an open source component, please don't put this here 😅

Reply:
I'm fine with removing it if you'd like me to.